avae/vis.py

```diff
@@ -173,8 +173,8 @@ def accuracy_plot(y_train, ypred_train, y_val, ypred_val):
 
 
 def loss_plot(epochs, train_loss, val_loss=None, p=None):
-    print("\n################################################################")
-    print("Visualising loss ...\n")
+    # print("\n################################################################")
```

Comment:
Why is this commented out?

Reply:
This is following the conversation above about print outs and verbose mode

Reply:
Ah. But all the other plotting printouts are still active so why only comment this single one out?

Reply:
because it prints out in every epoch and we were discussing that it is only necessary to see for debugging. if you think otherwise we can keep it in

Reply:
I actually find it useful to know what is visualised when, so these are not debug messages but useful feedback, especially as frequency of these can be controlled. Say you're visualising disentanglement every 50 epochs but reconstructions every 5 epochs, when I see the print flash past I can go and inspect it.

I see your point for the loss specifically because it is visualised every epoch anyway, but if `vis_los` is set to `False` then it doesn't visualise it at all, so it's useful to know this flag is active. So rather I'd prefer if these were controlled with verbosity as per conversation earlier instead of just commenting this out.

Reply:
yes verbosity as discussed above is the ideal way.

Reply:
So should we take the comment out for now and handle this properly in another PR? I'm worried if we leave it here, we'll forget to restore it when implementing verbosity.

Reply:
sure

Reply:
Thanks @crangelsmith ,so sorry for doubling the work. Please put back the prints. For now, I can just comment them out when running in terminal and put them back in before creating PR. Cheers

Reply:
I just realised that there was a full discussion before my comment, I think I didn't reload my page, apologies if my answer sounded repetitive. I'll add the print back.